### ISSUE TITLE
CSS files are generated twice with add_import_path 'sass'

### ISSUE THREAD

**Issue by jameswilson** [Compass/compass#1737]
After upgrading to sass 3.3 (from 3.2) and compass 1.0.0.rc.0  (from 0.12.6),  when I now run `bundle exec compass compile` I get css files generated twice:  once inside `./css`  and then the same files re-generated inside `./css/sass`.

I've looked around for this issue but both Google and this issue queue search turned up nothing.

I'll be happy to paste in my Gemfile and config.rb configuration if necessary, but for now I'll just paste in a diff of the changes in the config.rb files and Gemfile.lock to illustrate what changed since the update (not much has).

```
 DEPENDENCIES
-  breakpoint
-  compass
+  breakpoint (~> 2.4.2)
+  compass (~> 1.0.0.rc.0)
   compass-normalize
   compass-rgbapng
   compass-validator
@@ -107,9 +104,9 @@ DEPENDENCIES
   rb-fchange
   rb-fsevent
   rb-inotify (~> 0.9)
-  sass
-  sass-globbing
-  singularitygs
-  susy
-  toolkit
+  sass (~> 3.3)
+  sass-globbing (~> 1.1.0)
+  singularitygs (~> 1.2.1)
+  susy (~> 2.1.2)
+  toolkit (~> 2.5.2)
   yajl-ruby
```

and in config.rb:

```
-sass_options = {:debug_info => false, :line_comments => false}
+sass_options = {:debug_info => false, :line_comments => false, :sourcemap => true}
```

**Comment by jameswilson** [comment]
Complete config.rb file:

```
# Location of the theme's resources.
css_dir = "css"
sass_dir = "sass"
images_dir = "images"
generated_images_dir = images_dir + "/generated"
javascripts_dir = "js"

# Require any additional compass plugins installed on your system.
require 'compass-normalize'
require 'rgbapng'
require 'breakpoint'
require 'singularitygs'
require 'toolkit'
require 'susy'
require 'sass-globbing'

##
## You probably don't need to edit anything below this.
##

# You can select your preferred output style here (:expanded, :nested, :compact
# or :compressed).
output_style = :expanded

# To enable relative paths to assets via compass helper functions. Since Drupal
# themes can be installed in multiple locations, we don't need to worry about
# the absolute path to the theme from the server omega.
relative_assets = true

# Conditionally enable line comments when in development mode.
line_comments = false

# Output debugging info in development mode.
sass_options = {:debug_info => false, :line_comments => false, :sourcemap => true}

# Add the 'sass' directory itself as an import path to ease imports.
add_import_path 'sass'
```

**Comment by jameswilson** [comment]
The problem appears to be related to this line in config.rb:

```
# Add the 'sass' directory itself as an import path to ease imports.
add_import_path 'sass'
```

If i comment that line out and recompile, then there are no more duplicate css files generated.

Related issue on the Omega issue queue explains why the line was added: https://www.drupal.org/node/2177397

Apparently it just causes problems on newer versions of compass.

**Comment by chriseppstein** [comment]
The reason that you needed that extra `add_import_path` was a bug in ruby 2.0 and listen. It has been fixed in compass `1.0.0.rc.0`. You should take it out, but also, it does seem strange that this happened. I have fixed the bug.

**Comment by jameswilson** [comment]
Well that was fast, amazing work. Thank you for compass, and have a great weekend!
